index.bs

```diff
@@ -15,6 +15,10 @@ Markup Shorthands: markdown yes
 spec:webidl; type:dfn; text:resolve
 spec:webidl; type:dfn; text:new
 spec:infra; type:dfn; text:list
+spec:html; type:dfn; text:entangle
```

Comment:
I'm surprised these are needed; there are really other specs that define "create a new MessagePort object" and friends?

Well, hopefully we can eliminate these, even if it requires some HTML edits. We'll see.

Reply:
These are just not exported. I assumed we would probably end up modifying HTML to export them.